poetry/orb.yaml

```diff
@@ -17,15 +17,30 @@ jobs:
           Working directory for your package. Should point to a folder
           containing your pyproject.toml file.
         type: string
+      password:
+        default: $GEMFURY_TOKEN
+        description: >
+          The password used to authenticate the repository-name.
       python_version:
         default: latest
         type: string
+      repository-name:
```

Comment:
Looks like all other params use underscores rather than hyphens -- we should keep things consistent.

Reply:
This is no more, but there is a parameter in publish that has been changed to use an underscore. This change requires a major release.